### PR TITLE
Tweak browser column labels to card type, note type

### LINKED ISSUE #2749: Tweak browser column labels
https://forums.ankiweb.net/t/rename-the-column-of-the-browsers-table-card-note/35915

### PR DESCRIPTION
**Pull request by Gustaf-C**
This fixes #2749, unless I missed something.

### CODE REVIEW

File: ftl/core/browsing.ftl [hunk 1]
```diff
@@ -14,6 +14,7 @@ browsing-buried = Buried
 browsing-card = Card
 browsing-cards = Cards
 browsing-card-list = Card List
+browsing-card-type = Card Type
```
Comment (dae) [inline ftl/core/browsing.ftl:17]:
```suggestion
```

File: ftl/core/browsing.ftl [hunk 2]
```diff
@@ -59,6 +60,7 @@ browsing-no-flag = No Flag
 browsing-no-selection = No cards or notes selected.
 browsing-note = Note
 browsing-notes = Notes
+browsing-note-type = Note Type
```
Comment (dae) [inline ftl/core/browsing.ftl:63]:
```suggestion
```

File: rslib/src/browser_table.rs [hunk 3]
```diff
@@ -152,7 +152,7 @@ impl Column {
         match self {
             Self::Answer => tr.browsing_answer(),
             Self::CardMod => tr.search_card_modified(),
-            Self::Cards => tr.browsing_card(),
+            Self::Cards => tr.browsing_card_type(),
```
Comment (dae) [inline rslib/src/browser_table.rs:155]:
```suggestion
            Self::Cards => tr.card_stats_card_template(),
```

File: rslib/src/browser_table.rs [hunk 4]
```diff
@@ -161,7 +161,7 @@ impl Column {
             Self::Lapses => tr.scheduling_lapses(),
             Self::NoteCreation => tr.browsing_created(),
             Self::NoteMod => tr.search_note_modified(),
-            Self::Notetype => tr.browsing_note(),
+            Self::Notetype => tr.browsing_note_type(),
```
Comment (dae) [inline rslib/src/browser_table.rs:164]:
```suggestion
            Self::Notetype => tr.notetypes_notetype(),
```

**Review by dae — COMMENTED** [review]
Thank you! We can actually reuse some existing strings here, to avoid creating extra work for the translators.

**Comment by Gustaf-C** [comment]
Of course, I didn't think about that. My bad!

**Comment by dae** [comment]
Thank you!

**Pull request merged**